plugins/swf-backend/src/service/router.ts

```diff
@@ -68,7 +68,7 @@ export async function createRouter(
     '../../plugins/swf-backend/workflow-service/src/main/resources:/home/kogito/serverless-workflow-project/src/main/resources';
   const kogitoServiceContainer =
     config.getOptionalString('swf.workflow-service.container') ??
-    'quay.io/kiegroup/kogito-swf-devmode:1.40';
+    'quay.io/kiegroup/kogito-swf-devmode:latest';
```

Comment:
There is a risk that this will lead to port conflicts.

When we stop Backstage we have no code to automatically delete the `kogito-swf-devmode` container.

Should a newer version of the container be released when we run Backstage a new container may be instantiated with the later version using the same port... or I worry to much , as it'd be easily fixed by manually deleting the older container.